internal/store/store.go

```diff
@@ -489,6 +489,11 @@ func (s *Store) PushCompose(composeID uuid.UUID, bp *blueprint.Blueprint, checks
 	return nil
 }
 
+func (s *Store) PushRcmCompose(composeRequest ComposeRequest) error {
```

Comment:
This looks like it belongs in a different commit.

Reply:
You are right.